frontend/src/modules/genericeditor/components/GenericTranslationForm.js

```diff
@@ -59,7 +61,12 @@ export default class GenericTranslationForm extends React.Component<EditorProps>
             this.props.resetSelectionContent();
         }
 
-        this.focusInput(editor.changeSource !== 'internal');
+        const prevPropsWithoutUser = (({ user, ...rem }) => rem)(prevProps)
+        const propsWithoutUser = (({ user, ...rem }) => rem)(this.props)
+
+        if(!isEqual(prevPropsWithoutUser, propsWithoutUser)){
```

Comment:
I guess we can simplify this by comparing all props (without excluding the `user`).

Reply:
If there is a change in user-data isEqual(prevPropsWithoutUser, propsWithoutUser) will return false, and the editor gets focused.
As discussed, yesterday, we don't want to focus the editor even when props.user changes. Isn't this the case?

Reply:
Good point. Let's keep it that way.

The code to exclude user from props is pretty hard to understand. I wonder if we can make it more straightforward. Maybe we can use `delete`?